_episodes/04-higgs-search.md

```diff
@@ -458,17 +458,19 @@ len(stack_data_list_m4l)
 {: .output}
 
 To make more easy the data vs. MC final plot, we can define the following helper function that makes a histogram of the data and calculates the poisson uncertainty in each bin.
-When we want to make a plot that includes uncertainties we need to use the `plt.errorbar` function.
+When we want to make a plot that includes uncertainties we need to use the `ax.errorbar` function.
 
 ```python
 def plot_data(data_var, range_ab, bins_samples):
     data_hist, bins = np.histogram(data_var, range=range_ab, bins=bins_samples)
     print(data_hist, bins)
     data_hist_errors = np.sqrt(data_hist)
     bin_center = (bins[1:] + bins[:-1]) / 2
-    h0 = plt.errorbar(
+    fig, ax = plt.subplots()
+    h0 = ax.errorbar(
```

Comment:
seems like the assignment to `h0` is superfluous, right?

Reply:
for the function, unless we want to return h0, the h0= is superfluous, what do you think?

Reply:
exactly.

Though I feel it would actually be good practice to return the figure `fig` and get rid of the `plt.show()` instead. Should have the same effect

Reply:
done